ansible-tests/validations/controller-ulimits-check.yaml

```diff
@@ -0,0 +1,34 @@
+---
+- hosts: controller
+  vars:
+    metadata:
+      name: Check controller ulimits
+      description: >
+        This will check the ulimits of each controller.
+      groups:
+      - post-deployment
+    nofiles_default: 1025
+    nproc_default: 1025
+  tasks:
+  - name: Get nofiles limit
+    become: true
+    command: sh -c "ulimit -n"
+    register: nofilesval
+    changed_when: False
+  - name: Check nofiles limit
+    fail:
+      msg: >
+        nofiles is set to {{ nofilesval.stdout }}.  It should be higher.
```

Comment:
ok I will find out what we are recommending.  I don't believe we have a set recommendation other than "higher than 1024".